Add article about Pyvo @ EP (#93)
pyvec/blog.python.cz: content/2025-09-04_pyvo-na-ep.md

Title: Na Pyvo s EuroPythonem
Date: 2025-09-04 18:00:00
Author: Mia Bajić
Gravatar: miabajic.miabajic@gmail.com
About: Mia je vývojářka, místopředsedkyně EuroPython Society, a spoluorganizátorka pražského Pyva. 

Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.

> Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!).

Reviewer: ```suggestion
Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolina měla přednášku o tom, jak by si přála, aby vypadaly Python projekty z pohledu balíčkování, metadat a testovatelnosti. Jakožto vývojařka Fedora Linuxu každý rok integruje cca 4000 Python knihoven do distribuce, což jí dává poměrně široký záběr na svět tzv. upstreamu. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!).
```

> Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!s).

Reviewer 2: > kdo upstream udržuje 200 knihoven

Tohle mi úplně nedává smysl/nesedí. Cc @befeleme

Author: já jsem to takto pochopila z popisu přednášky, ale pokud máš návrh, jak to lépe říct, tak ho klidně přidej přes _Add a suggestion_ a akceptuju ho

Reviewer 2: Nechám to na @befeleme.
V práci knihovny udržuje v downstreamu a je jich cca 4 tisíce.

Author: aktualizováno, díky @befeleme!

> Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!s).

Author: ```suggestion
Z české komunity se zapojila spousta lidí, někteří jako dobrovolníci nebo přímo v organizačním týmu, další i na pódiu. Karolína měla přednášku o open source z pohledu někoho, kdo upstream udržuje 200 knihoven, praktický přehled toho, co by měl vědět každý open source maintainer. Lumír s Mirem tu navíc měli lightning talk o tom, jak správně vyslovovat PyPI (většina lidí říká PyPy, což je ale úplně jiný projekt!).
```

> ![lidé přednášející]({static}/images/ep2025.png)

Author: ```suggestion
![lidé přednášející]({static}/images/ep2025.png)
*Převzato z [EuroPython Flickr účtu](https://www.flickr.com/photos/europython/albums) s povolením organizátorů*
```

A při závěrečné session se ukázalo, kolik hudebníků se v komunitě sešlo: společně zahráli *Always Look on the Bright Side of Life*.

> ![lidé hrající na nástroje]({static}/images/ep20251.png)

Author: ```suggestion
![lidé hrající na nástroje]({static}/images/ep20251.png)
*Převzato z [EuroPython Flickr účtu](https://www.flickr.com/photos/europython/albums) s povolením organizátorů*
```

Shodou okolností vyšla konference právě na třetí středu v měsíci, tedy na den, kdy se v Praze pravidelně koná Pyvo. Tentokrát se tak lokální tradice spojila s EuroPythonem a výsledkem byl velký společný pokec u piva. Dorazilo kolem 200 lidí, což z něj dělá jedno z největších Pyv vůbec, stejně jako v roce 2023 na stejném místě. 

Pyvo proběhlo v hospodě Na Hradbách, tradičně oblíbeném místě účastníctva EuroPythonu, kousek od Kongresového centra. Přednášky nebyly, jen zahrádka, hospodská atmosféra a spousta rozhovorů.

> 

Author: ```suggestion

Na Pyvu se odehrála zajímavá událost. Framework Django letos slaví 20 let od svého vzniku, a tak organizace Django Software Foundation přinesla dort, o který se podělila s účastníky. Pyvo se dokonce objevilo na [oficiálním Django blogu](https://www.djangoproject.com/weblog/2025/sep/05/dsf-at-europython-2025-celebrating-20-years/).
```

Author: chceme toto zmínit?

Title: Na Pyvo s EuroPythonem
Date: 2025-09-04 18:00:00
Author: Mia Bajić
Gravatar: miabajic.miabajic@gmail.com
About: Mia je vývojářka, místopředsedkyně EuroPython Society, a spoluorganizátorka pražského Pyva. 

> Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnácet účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.

Reviewer: ```suggestion
Do Prahy se v červenci 2025 potřetí vrátila největší evropská Python konference, EuroPython. Do Kongresového centra dorazilo patnáct set účastníctva z celého světa a na programu nechyběly přednášky, workshopy, summity, komunitní stánky ani hallway tracky.
```
